src/generators/generate-models.ts

```diff
@@ -150,6 +174,8 @@ async function main() {
             }
         }
 
+        console.log('Generated models for tables:', tables);
+
```

Comment:
I assume this is intentional but not certain

Reply:
It is intentional! I'm sure we can find a better styling to how we convey what classes were created, though, or a summary thereof.